2024FALL-SWPP/team-project-for-2024-fall-swpp-team-16
# Feat/Retry button and logic

## Description
Added Reset Stage button to Asset and `SettingsModalManager.cs`to reset stage, adjusted StageScene and Stage 1,3,6 to work with this change

## Related Issue

#72 

## Changes Made

- [ ] Added `1. Retry` UI asset to `04.Images/Sprites/UI` 
- [ ] Added public Retry button to `SettingsModalManager Script.cs`, which must be assigned in inspector (with `1. Retry` in above change). The UI should be formatted nicely next to the existing Exit Game button
- [ ] Changed 'Exit Game' button to not end game but open StageScene in  `SettingsModalManager.cs`
- [ ] Added EventSystem and adjusted settings modal to stage 1,3 6 and StageScene

## How to Test This Branch
- Open StageScene or stage 1, 3 or 6. Press esc to open settings modal and check that 'retry' button works. check that 'exit game' leads to a working scenestage. 
- Could also try applying changes to other stages

## Screenshots (if applicable)


## Additional Notes
